services/Player.js

```diff
@@ -0,0 +1,80 @@
+import db from '../db'
+
+class Player {
+  pickRandomlySong() {
+    return db
+      .select('id')
+      .from('songs')
+      .limit(10)
```

Comment:
There really isn't any randomisation going on here :)
Also, you could provide input parameter for number of songs in this method, and use it in `limit(numberOfSongs)` call instead of hardcoding 10